dashboard/db/schema.rb

```diff
@@ -385,6 +385,7 @@
   end
 
   add_index "plc_user_course_enrollments", ["plc_course_id"], name: "index_plc_user_course_enrollments_on_plc_course_id", using: :btree
+  add_index "plc_user_course_enrollments", ["user_id", "plc_course_id"], name: "index_plc_user_course_enrollments_on_user_id_and_plc_course_id", using: :btree
   add_index "plc_user_course_enrollments", ["user_id"], name: "index_plc_user_course_enrollments_on_user_id", using: :btree
```

Comment:
you no longer need this index (mysql can use the prefix of a multiple column index -- so it can use the above index for queries on just user_id)

I think in general if you are removing the user_id index you want to do that after creating the user_id/course_id index (so that the db can use the user_id index to create the user_id/course_id index), for this it doesn't matter because you don't have much data